### PR TITLE
Fix Gradle docs about native generation and suspend debug instructions

### PR DESCRIPTION
**Pull request by gastaldi**
@geoand please review

### CODE REVIEW

File: docs/src/main/asciidoc/gradle-tooling.adoc [hunk 1]
```diff
@@ -280,7 +280,7 @@ An additional system property `suspend` can be used to suspend the JVM, when lau
 
 [TIP]
 ====
-You can also run a Quarkus application in debug mode with a suspended JVM using `./gradlew quarkusDev -Ddebug` which is a shorthand for `./gradlew quarkusDev -Ddebug=true`.
+You can also run a Quarkus application in debug mode with a suspended JVM using `./gradlew quarkusDev -Dsuspend=true -Ddebug` which is a shorthand for `./gradlew quarkusDev -Dsuspend=true -Ddebug=true`.
```
Comment (gastaldi) [inline docs/src/main/asciidoc/gradle-tooling.adoc]: I couldn't make it work with simply `-Dsuspend`

Comment (geoand) [reply docs/src/main/asciidoc/gradle-tooling.adoc]: That's odd, but ok I guess 😆

**Review by geoand — APPROVED** [review]

**Comment by geoand** [comment]
Thanks @gastaldi!

**Comment by gastaldi** [comment]
I made the tip a separate topic, since it the tip is not related to the parent topic

**Pull request merged**